Week-7/Question 3/3.js

```diff
@@ -0,0 +1,32 @@
+/* Question
+Given an array of size N containing only 0s, 1s, and 2s; sort the array
+in ascending order.
+    Example 1: Input: N = 5 , arr[]= {0 2 1 2 0}
+    Output:0 0 1 2 2
+ */
+
+//! TC->O(N), SC->O(1)
+var sortColors = function (nums) {
+  //* initial starting index
+  let index = 0;
+
+  //* positioning all zeroes into place
+  for (let i = 0; i < nums.length; i++) {
+    if (nums[i] === 0) {
```

Comment:
When comparing something with 0 you can use if (!nums[i]) only as NOT Operator will take 0 as false and whole condition as True